docs/install.md

```diff
@@ -0,0 +1,228 @@
+# Installing, Upgrading, and Removing
+
+Users can choose between operating systems of MacOS, Windows, and Linux for Chef InSpec.
+
+## Install Chef InSpec
+
+You can download the latest [Chef InSpec package](https://downloads.chef.io/inspec) relevant to your operating system. 
+
+Alternatively, Chef InSpec can be installed via script according to your operating system and method through the methods below.
+macOS install options include [Homebrew](https://brew.sh/), CLI, and Chef Habitat. Windows install options include Installer, Powershell, and Chef Habitat. Linux install options covers Ubuntu, Red Hat Enterprise Linux, and SUSE Linux Enterprise Server.
+
+### macOS
+
+#### Homebrew
+
+Chef InSpec is available as a standalone [Homebrew](https://brew.sh/) package. Run the following command in your terminal to install Chef InSpec:
+
+``` 
+$ brew cask install chef/chef/inspec
+```
+
+While this command is running, you may be prompted to enter your macOS user account password for installation to complete.
+
+#### CLI
+
+You can download Chef InSpec via script:
+```
+curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+```
+
+#### Habitat
+
+Use Chef Habitat to install Chef InSpec, or include Chef InSpec in your own Chef Habitat packages. Chef InSpec is available [on the Habitat Builder Depot](https://bldr.habitat.sh/#/pkgs/chef/inspec).
```

Comment:
Looks to me like this is NOT a valid option for mac.

Reply:
Habitat for Mac runs Linux under the covers, so it technically is a valid option for Mac even though the platform isn't listed on bldr.

Reply:
Not an issue anymore :)